packages/components/package.json

```diff
@@ -96,10 +96,12 @@
     "@wapps/storybook-addon-state": "0.0.4",
     "classnames": "^2.2.6",
     "@deriv/shared": "^1.0.0",
+    "@deriv/translations": "^1.0.0",
```

Comment:
I thought we decided not to add translations to components? as previously discussed.